README.md

```diff
@@ -51,6 +51,9 @@ Sample usage:
       // overrides project IntermediaryOutputPath
       intermediateDir = 'build/msbuild/obj"
       
+      // temporary directory where the plugin will extract the project file parser executable

```

Comment:
I'd rather avoid making this configurable, it shouldn't be (many gradle plugins download&execute binaries, I've never seen on make the location configurable)